src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -161,21 +161,16 @@ CHIP_ERROR writeFabricsIntoFabricsListAttribute()
     return err;
 }
 
-/*
- * Look at "Temporary flow for fabric management" comment above for current fabric management flow.
- * To retrieve the current fabric, we retrieve the emberAfCurrentCommand()->source which should be set
- * to the commissioner node Id, which we are temporarily using as the fabricId.
- * We should also figure out how to retrieve the device nodeId and vendorId if we can so that we use multiple
- * fields to find the current fabric. Once addOptCert and fabric index are implemented, remove all this and use fabricIndex.
- */
-
 static FabricInfo * retrieveCurrentFabric()
 {
-    uint64_t fabricId = emberAfCurrentCommand()->SourceNodeId();
-    // TODO: Figure out how to get device node id so we can do FindFabricForNode(fabricId, nodeId)...
-    emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: Finding fabric with fabricId 0x" ChipLogFormatX64 ".",
-                   ChipLogValueX64(fabricId));
-    return GetGlobalFabricTable().FindFabricForNode(fabricId);
+    if (emberAfCurrentCommand()->source == nullptr)
```

Comment:
This does not necessarily need to happen now.  But I believe the information currently coming from`emberAfCurrentCommand()` could (and eventually should) come from `app::CommandHandler * commandObj`, which is available in all of the callers to `retrieveCurrentFabric()` I can find.

Reply:
https://github.com/project-chip/connectedhomeip/issues/8906